crates/data-model/Cargo.toml

```diff
@@ -28,3 +28,5 @@ ruma-common.workspace = true
 mas-iana.workspace = true
 mas-jose.workspace = true
 oauth2-types.workspace = true
+# Emails
+lettre.workspace = true
```

Comment:
Can you put that in the list of external crates up there without the comment for consistency?